mordred/config.py

```diff
@@ -247,6 +247,16 @@ def general_params(cls):
                     "optional": True,
                     "default": "git",
                     "type": str
+                },
+                "kibiter_url": {
```

Comment:
Wouldn't it better to have a new section for Kibiter, and have these two new options in it?

Reply:
I think that probably we need to change the `panel` section name with `kibiter` name. All the params in the `panel`section are related to kibiter. But this change breaks all current config files so I prefer to do it once the platform is more stable without so many changes. This is why I have added those new params to the `panel`section.

Reply:
OK, now I understand. Looking at the parameters, they are more like parameters of the dashboard than parameters of the service (in the sense that to produce the same dashboard on a different deployment, those parameters wouldn't change). That´s why I still see a reason for having a separate kibiter section.

But I see your point. So, you decide. If you prefer, keep it like it is now.

Reply:
Yes, I prefer to maintain to keep as it is now and in a next iteration rename the full section to Kibiter. So I think we are ready to merge this PR? l let that you do it! Thanks @jgbarah